openid-4-verifiable-presentations-1_0.md

```diff
@@ -581,12 +581,24 @@ The following is a non-normative example of the Authorization Response that is s
     state=eyJhb...6-sVA
 ```
 
-If the Response Endpoint has successfully processed the request, it MUST respond with HTTPS status code 200. 
+The following is a non-normative example of an Authorization Error Response that is sent as an HTTP POST request to the Verifier's Response Endpoint:
+
+```
+  POST /post HTTP/1.1
+    Host: client.example.org
+    Content-Type: application/x-www-form-urlencoded
+
+    error=...&
+    error_description=...&
```

Comment:
Sorry, I took another quick look and spotted two minor things  - it's nice if the examples are actual examples as far as possible (sometimes we use '...' but we should try and only do that for things that are too large to include in full)

```suggestion
    error=invalid_request&
    error_description=unsupported%20client_id_scheme&
```